tests/lib/snaps/test-snapd-service/meta/hooks/configure

```diff
@@ -6,9 +6,10 @@ COMMAND=$(snapctl get command)
 if [ "$COMMAND" != "" ]; then
     if [ "$COMMAND" = "restart" ]; then
         snapctl restart test-snapd-service.test-snapd-service
-        snapctl restart test-snapd-service.test-snapd-other-service
     else
         snapctl "$COMMAND" test-snapd-service.test-snapd-service
     fi
 fi
 sleep 3
+
```

Comment:
Can you document why this is significant so that it doesn't go away by accident later.